### DISCUSSION TITLE
Tab Size - Code Format

### DISCUSSION THREAD

**Discussion by babakfp** [laravel/framework#40548] [General] [answered]
- Laravel Version: 8.75
- PHP Version: 8.1.0

### Description:
Hi
My editor's tab size is set to `2` and Laravel's source code is set to `4`. So, I'm seeing 2 tabs instead of 1.
Thanks
![Screenshot 2022-01-21 133221](https://user-images.githubusercontent.com/44144724/150603572-f7824736-5aeb-45ae-8343-c3eaba6690c3.png)

### Steps To Reproduce:
`composer create-project laravel/laravel`

**Comment by babakfp** [comment]
Why is this converted into a "discussion" when there is nothing to discuss?

**Comment by eusonlito** [comment, marked as answer]
Set your editor to 4 spaces, as PHP PSR-12 rules say:

https://www.php-fig.org/psr/psr-12/

> 2.4 Indenting
> Code MUST use an indent of 4 spaces for each indent level, and MUST NOT use tabs for indenting.

**Reply by babakfp** [reply]
So, this is a consistent way that everyone follows?

**Reply by eusonlito** [reply]
Yes, are the standards about code style (and more) on PHP: https://www.php-fig.org/psr/

PSR-1: Basic Coding Standard
PSR-12: Extended Coding Style Guide

**Reply by babakfp** [reply]
Makes sense
